packages/cw-orch-mock/src/core.rs

```diff
@@ -370,15 +411,16 @@ impl BankQuerier for Mock {
 impl BankSetter for Mock {
     fn set_balance(
         &mut self,
-        address: &Addr,
+        address: impl Into<String>,
```

Comment:
I used to really like the `impl Into<String>` but now that addresses (`Addr`) must be valid it is too easy to mis-use IMO. 

Opinions?

Reply:
Yeah ... I think that we should avoid using Addr::unchecked() as much as possible now ! Just like in contracts. 
So getting an Addr is mandatory here I think